README.md

```diff
@@ -107,7 +107,7 @@ We are agnostic about how you incorporate external data into the template. There
     * The download can be incorporated into the SCons build. We either execute a program to transfer data (e.g., `rsync` or `rclone`) using our "Anything builder" or from within a script executed by one of our other custom builders. These methods have the benefits of automation and dependency tracking, but they can introduce idiosyncratic errors if the download steps are prone to failure.
     * Regardless of the download method, the path to the dataset should be added to `config_global.yaml` and `.gitignore` if it is stored within the repository and to `config_user.yaml` if it is stored elsewhere. 
 
-* Any pointers to directories you store under the `external` key in either configuration yaml file will have their contents checked and recorded at runtime. We complete the check using our `record_dir` function and store its contents in `release/state_of_external.log`.
+* Any pointers to directories you store under the `input` key in either configuration yaml file will have their contents checked and recorded at runtime. We complete the check using our `record_dir` function and store its contents in `release/state_of_input.log`.
```

Comment:
Wait, didn't we go through a long and bitter (not really) debate about whether to call this state_of_external.log vs state_of_inputs.log?

Reply:
I also recall that we settled on `input`. I'm fine with that then, just confused as to why it wasn't so already.

Reply:
Ahhh, that would be because I forgot to update the readme. I caught it in this task and figured we should fix the error.